polaris-service/src/test/java/org/apache/polaris/service/admin/PolarisOverlappingCatalogTest.java

```diff
@@ -110,101 +111,60 @@ private Response createCatalog(
     }
   }
 
-  @Test
-  public void testBasicOverlappingCatalogs() {
-    Arrays.asList(false, true)
-        .forEach(
-            initiallyExternal -> {
-              Arrays.asList(false, true)
-                  .forEach(
-                      laterExternal -> {
-                        String prefix = UUID.randomUUID().toString();
-
-                        assertThat(createCatalog(prefix, "root", initiallyExternal))
-                            .returns(Response.Status.CREATED.getStatusCode(), Response::getStatus);
-
-                        // OK, non-overlapping
-                        assertThat(createCatalog(prefix, "boot", laterExternal))
-                            .returns(Response.Status.CREATED.getStatusCode(), Response::getStatus);
-
-                        // OK, non-overlapping due to no `/`
-                        assertThat(createCatalog(prefix, "roo", laterExternal))
-                            .returns(Response.Status.CREATED.getStatusCode(), Response::getStatus);
-
-                        // Also OK due to no `/`
-                        assertThat(createCatalog(prefix, "root.child", laterExternal))
-                            .returns(Response.Status.CREATED.getStatusCode(), Response::getStatus);
-
-                        // inside `root`
-                        assertThat(createCatalog(prefix, "root/child", laterExternal))
-                            .returns(
-                                Response.Status.BAD_REQUEST.getStatusCode(), Response::getStatus);
-
-                        // `root` is inside this
-                        assertThat(createCatalog(prefix, "", laterExternal))
-                            .returns(
-                                Response.Status.BAD_REQUEST.getStatusCode(), Response::getStatus);
-                      });
-            });
+  @ParameterizedTest
+  @CsvSource({"true, true", "true, false", "false, true", "false, false"})
```

Comment:
nit: Rather than CsvSource, I would rather use `MethodSource` or `ArgumentsSource`

Reply:
At the risk of starting a bike-shedding discussion :) I'd say that IMHO `@CsvSource` is more readable than `@MethodSource` for simple parameter tuples. More complex parameters would certainly warrant a `@MethodSource`.

Reply:
I think we're good here. We can improve when necessary.